Sources/Orbit/Components/Badge.swift

```diff
@@ -6,20 +6,22 @@ import SwiftUI
 /// They can be updated when a status changes, but they should not be actionable.
 ///
 /// - Note: [Orbit definition](https://orbit.kiwi/components/badge/)
-public struct Badge: View {
+public struct Badge<LeadingIcon: View, TrailingIcon: View>: View {
```

Comment:
Commenting further after seeing all changes - I like `Prefix` and `Suffix` more where applicable. I still don't think we should use `Icon` if it can actually be any view.

Reply:
This is based on orbit.kiwi + figma naming. Even though there can technically be anything, here it is still semantically an icon that explains the label, while a ListChoice content is really "any custom content".

<img width="279" alt="image" src="https://github.com/kiwicom/orbit-swiftui/assets/35844477/2abc1a14-7be2-469d-b087-34dcb98ece19">